### DISCUSSION TITLE
Background choice

### DISCUSSION THREAD

**Discussion by 9atrotte** [YelloElefant/13IA#30] [Ideas]
lets talk about the background theme

**Comment by 9dsklena** [comment]
Change the background, the current gray block is a bit bland and boring and does not look right for the website. A background similar to this may look good.
![image](https://github.com/9atrotte/13IA/assets/107901450/7a7cb0e1-1a00-44fe-8547-ee3412eb889b)

**Comment by 9atrotte** [comment]
footer with that and white in the middle maybe
